### PR TITLE
feat: add ifftn to jax frontend

### LINKED ISSUE #22389: ifftn

### PR DESCRIPTION
**Pull request by abdllahdev**
Close #22389

### CODE REVIEW

File: ivy/functional/frontends/jax/numpy/fft.py [hunk 1]
```diff
@@ -27,3 +27,9 @@ def fftshift(x, axes=None, name=None):
     roll = ivy.roll(x, shifts, axis=axes)
 
     return roll
+
+
+@to_ivy_arrays_and_back
+def ifftn(a, s=None, axes=None, norm=None):
+    a = ivy.asarray(a, dtype=ivy.complex128)
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/jax/numpy/fft.py:34]: As it is mentioned in the [Docs](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.fft.ifftn.html) no need to explicitly convert the input to complex, but input itself should be a complex type, this line can be omitted and we specify supported dtypes as complex using `with_supported_dtypes` decorator

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @abdllahdev 

Check the comments below
Thanks

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @abdllahdev 

Thanks for updating the requested changes, would you resolve the linting issues as the lint check is failing you can refer to this [Pre-Commit](https://unify.ai/docs/ivy/overview/deep_dive/formatting.html#pre-commit) guide

Thanks

**Comment by abdllahdev** [comment]
Hey @hirwa-nshuti, the pre-commit hooks had a problem before but now it's working. I pushed the new changes and it should be fine now. Thanks

**Comment by abdllahdev** [comment]
Hey @hirwa-nshuti, Can you please check this PR. There are merge conflicts, formating is good and everything should be good to merge. Thanks

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
If you check on bottom section of the PR comments,
You will find that there are merge conflicts and to resolve them you can follow this guide [managing your fork](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#managing-your-fork)

After that we can be able to run the tests on CI.
Thanks

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @abdllahdev 
would you remove the changes that you added to the Array API submodule

Thanks!

**Comment by Sarvesh-Kesharwani** [comment]
Closing the PR for the time being, since there has been no activity for more than 2 weeks.

**Pull request closed without merging**